partners/README.md

```diff
@@ -0,0 +1,108 @@
+# Partner Tekton Task submission process
+
+Partners can submit their Tekton Task in [build-definitions](https://github.com/redhat-appstudio/build-definitions) repository by sending a pull request.
```

Comment:
https://github.com/redhat-appstudio/book/blob/main/ADR/0021-partner-tasks.md should be linked somewhere here.

Reply:
Added one extra line in this section referring the ADR `For more information, please refer the ADR`